save-backend/build.gradle.kts

```diff
@@ -4,9 +4,10 @@ import  org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     kotlin("jvm")
+    id("org.jetbrains.kotlin.plugin.jpa") version Versions.kotlin
```

Comment:
`jpa` plugin needs to be applied in save-common now

Reply:
and `all-open` too

Reply:
Added